js_modules/dagit/packages/core/src/ui/SectionedLeftNav.tsx

```diff
@@ -101,7 +102,7 @@ export const SectionedLeftNav = () => {
   // Sort repositories alphabetically, then move empty repos to the bottom.
   const sortedRepos = React.useMemo(() => {
     const alphaSorted = [...visibleReposAndKeys].sort((a, b) =>
-      a.key.toLocaleLowerCase().localeCompare(b.key.toLocaleLowerCase()),
+      a.label.toLocaleLowerCase().localeCompare(b.label.toLocaleLowerCase()),
```

Comment:
Use `key` for the URI-encoded version, which is what we use for the expansion state. Add a `label` value for alphabetization purposes.